Python/Gmeet py Bot/Gmeet_buddy.py

```diff
@@ -0,0 +1,105 @@
+# Autour : Yuvraj Kadale
```

Comment:
No need to add GSSoC here.
Add a Author section in README rather than here on script